app/modules/users/parameters.py

```diff
@@ -208,4 +206,51 @@ def replace(cls, obj, field, value, state):
                 #     # Access granted
                 #     pass
 
+        if field == User.profile_fileupload_guid.key:
+            value = cls.add_replace_profile_fileupload(value)
+
         return super(PatchUserDetailsParameters, cls).replace(obj, field, value, state)
+
+    @classmethod
+    def remove(cls, obj, field, value, state):
+        from app.modules.fileuploads.models import FileUpload
+
+        if field == User.profile_fileupload_guid.key:
+            if obj.profile_fileupload_guid:
+                fup = FileUpload.query.get(obj.profile_fileupload_guid)
```

Comment:
this is as much for my own education as possible change, but can this now be replaced with `fup = obj.profile_fileupload` instead, since you made the change with the reference?

Reply:
Yes I think so, I'll change it :+1: